rest-service/manager_rest/config.py

```diff
@@ -16,6 +16,7 @@
 import os
 import yaml
 
+import jaeger_client as j_client
```

Comment:
I missed this, there is a confusion here yaml is python standard lib and json is